libraries/AP_Tuning/AP_Tuning.cpp

```diff
@@ -228,6 +228,15 @@ void AP_Tuning::check_input(uint8_t flightmode)
     need_revert |= (1U << current_parm_index);
     set_value(current_parm, new_value);
 
+    if ( fabsf(new_value-old_value) > (0.05 * old_value) ) {
+        old_value = new_value;
+        GCS_SEND_TEXT(MAV_SEVERITY_INFO, 
+            "Tuning %s%s%0.4f", 
```

Comment:
if the intention is this comes out as speech in mavproxy it needs to start with "Tuning: " or an "@"